Merge community 2.4 (2018-05-07) with Carto branch

pramsey opened the pull request:
This takes current community state and brings it in. This fixes some issues, in particular one that came up on twitter regarding new behaviour in ST_Segmentize(geography).

pramsey committed:
Replace project-and-entend logic with 
bisect-and-recurse in geography segmentization.
Preserves "mostly equal" segment lengths, and 
should be more numerically stable. 
Backport to 2.4.
References #3667


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16094 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add missing release notes for ST_Segmentize geography fix.
References #3667 for PostGIS 2.4.2

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16098 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix spacing

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16099 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add missing 2.3.4 upgrade path (as provided by Muhammad Usama)
Also remove 2.3 and 2.4 dev releases and add missing 2.2.6
Closes #3926 for PostGIS 2.4.2

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16102 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
note about enhanced feature in reverse_geocode when zcta5 data is loaded.

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16108 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Prepare for 2.4.2


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16112 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
push version numbers forward


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16114 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add the freebsd bot scripts -- the svn hook change should make these fire right away

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16122 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Explicitly specify $(PERL) to call create_unpackaged.pl otherwise at least on freebsd it fails
Also get rid of 9.1 check, we don't support 9.1 anymore.
References #3938 for PostGIS 2.4.3


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16123 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Support google line encodings that include a backslash character (References #3713)



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16131 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Turn off interrupt tests in configure by default.
Also added a section in configure output to note if interrupt tests are enabled or not
Closes #3827 for PostGIS 2.4.3

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16135 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
use new osgeo project logo

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16139 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
add in verbose and logic to drop test database if it exists.  Switch to using BRANCH instead of TAG

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16146 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Update SRS to EPSG 9.2

Patch by Even Rouault
Ref #3944 for 2.4 branch

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16157 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Manual fixes from PostGIS users (not many left now) to EPSG file
References #3944


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16159 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Use original vertex values where available in contructing new pointarray (references #3941)



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16163 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Bring epsilon fix to minimum bounding circle back to 2.4 (closes #3930)


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16166 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Change syscache stats lookup to use STATRELATTINH key and three parameters in all cases for 2.4 branch. Closes #3904


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16176 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
copy over 2.3.4 and 2.3.5 notes

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16185 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
switch to have version of libs used be defined in script rather than the jenkins job

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16204 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
Drag in authors mapping from trunk.



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16211 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
Fix KMeans initialization issue that lost clusters sometimes.

Closes #3965
Closes https://github.com/postgis/postgis/pull/179


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16213 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Upgrade SFCGAL too when requested

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16239 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Do not assume CREATE OPERATOR CLASS needs to end with ');'

A single ending semicolon is enough to terminate the statement

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16240 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Simplify syntax for brin opclasses so upgrade script recognizes it

Fixes #3956 in 2.4 branch

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16241 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
Encoded Polyline of EMPTY
Pull in from trunk

Closes #3982


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16264 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
2.4 fix, st_transform does not qualify query to spatial_ref_sys (references #3975)


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16271 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
change to hard-code GDAL, GEOS, and MAKE_EXTENSION and dump so easier for people to set without changing the job.

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16273 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
more moving of variables to script from job

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16274 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
set to  a GEOS debbie doesn't have.  Change to GEOS 3.7.0dev

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16275 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
change to valid GDAL version

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16276 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
revert setting GDAL and GEOS, forgot pg launch script needs these which we don't have in repo.  Turn on DUMP_RESTORE testing

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16277 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add release notes to docs in prep for release. Minor reword in NEWs

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16279 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix typo in release version it's 2.4.3 not 2.4.8

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16280 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
revise make_dist.sh to allow overriding the package name, and create debbie make dist script that calls the main one
References #3985 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16284 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix typo in make_dist.sh 
Add doc building, 
update How To release to reflect tar ball and docs start building as soon as work is committed.
References #3985 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16288 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Change release version to 2.4.3rc1

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16289 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix missed edit.
References #3985 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16292 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
in prep for release Wednesday, flip back to dev, stamp expected date of release in news
Add 2.3.6 upgrade path

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16294 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
with_interrupt_tests using non-standard extension
Closes #3988 for PostGIS 2.4.3

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16298 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
update ChangeLog
flip Version.config to rc2
revise make dist to output branch reading from

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16299 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
more clearifying notes on HOWTO_RELEASE 
change version to 2.4.3 in prep for tagging tomorrow or Wednesday.

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16301 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Get rid of SVN number and revise to not build local html except for trunk

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16305 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
flip version to 2.4.4dev, add 2.4.3 to upgrade versions
add timestamp to dev docs 
closes #3991 for PostGIS 2.4.4dev

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16315 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Backport support for Pg11 to 2.4 branch. Closes #3946


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16320 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Updates news re pg11 support


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16321 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Use PKG_PROG_PKG_CONFIG macro from pkg.m4 to detect pkg-config

Patch by Bas Couwenberg
Closes #3992 in 2.4 branch (for 2.4.4)

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16325 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Move PgSQL 11 support note to 2.4.4
It was put in after 2.4.3 was already tagged and tarred.

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16328 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Forgot to add pkg.m4 macro in r16324

See #3992

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16336 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Have Debbie test all available upgrades

References #2676 for 2.4 branch

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16346 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
change scripts back to versions of things to compile against (GDAL, PROJ, SCGAL, GEOS) being set at job level until we have the stop and start postgres in repo as well.

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16347 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
backport run_test.pl fixes from trunk

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16350 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
[drone] Use trisquel2 image, test all pgsql versions and upgrades

The new docker image also uses newer GEOS (3.6.2).
References #3813 for 2.4 branch
NOTE: could not use -Werror in this branch as it would fail

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16353 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Set availability of support function for KNN index

Fixes upgrade from 2.1 to 2.2+
Closes #3978 in 2.4 branch (2.4.4)

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16358 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Avoid memory exhaustion when building a btree index (references #4004)
From Edmund Horner



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16361 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
lwpoly_construct_circle: Avoid division by zero (Raúl Marín Rodríguez)
References #4003


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16362 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
backport regress changes from 2.5

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16397 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add Raúl Marín Rodríguez to developers list

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16398 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Lowercase Algunenano

All LDAP account names are lowercase

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16399 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
[gitlab-ci] Switch to PostgreSQL 9.6 and libjson-c-dev

Fixes gitlab-ci build for 2.4, references #4023

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16401 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
Fix cast from box3d to geometry

Patch by Matthias Bay

Closes https://github.com/postgis/postgis/pull/214
Closes #4020



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16406 b70326c6-7e19-0410-871a-916f4a2858ee

strk committed:
Do not include private header for json-c >= 0.13

Patch by Björn Esser

Closes #3942 for 2.4 branch

(also fixes NEWS section corrupted by recent commit)

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16417 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Use more precise bounds when testing overlap of temporal ranges.
For 2.4, references #4025



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16420 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Fix memory corruption in case where lexer is interrupted mid-process
References #4017
For 2.4 branch


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16424 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Update lexer generated file
References #4017


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16425 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Add logic to show git log timestamp in UTC instead of Unix timestamp and put in svn revision number
References #4047 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16470 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
variable typo
References #4047 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16472 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
this for sure will fix revision # this time I tested before hand
References #4047 for PostGIS 2.4


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16474 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Seems sometimes the postgis_svn_revision.h gets wiped out, so end up with no svn number. 
Explicitly call svn_repo_revision.pl to ensure we have an svn number to read
References #4047 for PostGIS 2.4


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16477 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
schema qualify geography functions (including ST_Buffer)
References #4052 for PostGIS 2.4.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16482 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
change to use GDAL 2.2.4, 3.6.2 (for packaging) but keep 3.7.0dev for regress testing

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16484 b70326c6-7e19-0410-871a-916f4a2858ee

dbaston committed:
Avoid dropping SRID in ST_ClusterIntersecting

References #4055



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16491 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
note in docs about soft upgrade that legacy.sql must be rerun for each minor.
Closes #2960 for PostGIS 2.4.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16495 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
change sorting of tickets to be nuermic by ticket number.  Add in missing tickets

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16496 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
upgrade support for PostgreSQL 11
closes #4044 for PostGIS 2.4.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16498 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
ST_CPAWithin precision fix

Closes #4032


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16500 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Fix crasher in raster when fed an empty raster (note was tempted to change the def of empty raster to include raster with no bands but not sure where else that is used, so just revised the clip check to not try to clip if no bands).
References #3055 for PostGIS 2.4.4, 
also minor doc fix references #2925 for PostGIS 2.4.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16504 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Update in prep for 2.4.7 

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16517 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Fix typo in release notes

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16519 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix date typo in release notes

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16520 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Flip version to 2.4.5dev, revise winnie to be able to use source_folder if passed in (will allow tagged builds to be built from dev branch for testing)

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16527 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
fix path setting

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16529 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
switch to test against GEOS 3.6.2

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16532 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
missed some spots where source_folder variable is used

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16533 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
revise debbie to build against master sfcgal  
References #4068 for PostGIS 2.4

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16534 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Disable sfcgal testing until sfcgal 1.3.3 / CGAL 4.11 issue is resolved.
References #4068 for PostGIS 2.4


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16535 b70326c6-7e19-0410-871a-916f4a2858ee

robe2 committed:
Disable sfcgal in make dist
References #4068 for PostGIS 2.4



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16536 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Guard against lookup failure, to avoid segfault in internally called case (closes #4067)



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16543 b70326c6-7e19-0410-871a-916f4a2858ee

Komzpa committed:
ST_ClusterKMeans crash on NULL and EMPTY

Closes #4071



git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16550 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
ensure that polygons are CW coming out of the ASMVTGEOM process
Closes #4079


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16558 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Update NEWS re #4079


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16559 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Emit a cancellation error code on GEOS interruptedExceptions, for consistency
with internal interruptions.
Closes #4070


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16562 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Delay freeing input until processing is complete. From lucasvr.
Closes #3980


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16564 b70326c6-7e19-0410-871a-916f4a2858ee

pramsey committed:
Merge branch 'svn-2.4' of github.com:postgis/postgis into svn-2.4-cartodb-merge

pramsey committed:
Clean up build and merge issues

Commit:
Fixes bounding box handling in twkb_input #4089

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16573 b70326c6-7e19-0410-871a-916f4a2858ee

Algunenano commented:
I don't get it Github, this merges cleanly... I'll try to rebase everything to see if it does and the tests are run.

NVM. I didn't notice that it was setup as Rebase and merge vs Create merge commit

Commit:
Update to current 2.4 (2018-05-17)

Commit:
Backport PG 11 compatibility

Closes #4090
Closes https://github.com/postgis/postgis/pull/242


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16576 b70326c6-7e19-0410-871a-916f4a2858ee

Algunenano committed:
Update to current 2.4 (2018-05-18)

strk committed:
Survive to big max deviation values passed to ST_CurveToLine

When using "max-deviation" tolerance type, passing a tolerance
bigger than twice the radius of any arc resulted in entering
an infinite loop, only limited by availability of RAM.

This commit fixes the bug by being careful in what's fed to
acos()...

Includes a unit test.
Closes #4031 in 2.4 branch (2.4.5dev)

git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16577 b70326c6-7e19-0410-871a-916f4a2858ee

Commit:
Fix PG 9.5 build in 2.4

References #4090
Applies the same fix as #3950
Closes https://github.com/postgis/postgis/pull/243


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16578 b70326c6-7e19-0410-871a-916f4a2858ee

Commit:
Fix bug in lwgeom_median and avoid division by zero

References #3997
Closes https://github.com/postgis/postgis/pull/244


git-svn-id: http://svn.osgeo.org/postgis/branches/2.4@16579 b70326c6-7e19-0410-871a-916f4a2858ee

Algunenano committed:
Update to current 2.4 (2018-05-21)
